docs/getting-started/integrating-codacy-with-your-git-workflow.md

```diff
@@ -0,0 +1,56 @@
+---
+description: Configure Codacy to block merging pull requests that don't meet your quality standards.
+---
+
+# Integrating Codacy with your Git workflow
+
+Once you've [customized your setup](customizing-your-setup.md) to best match your needs, you can integrate Codacy with your Git workflow to display analysis results and code coverage on your Git provider and block problematic pull requests.
+
+To integrate Codacy with your Git workflow, follow these steps:
+
+1.  [Configuring quality settings](#configuring-quality-settings)
+1.  [Activating the Git provider integration](#git-provider-integration)
+1.  [Using code coverage as a quality gate](#coverage-gate)
+1.  [Blocking merging pull requests](#blocking-pull-requests)
+
+## 1. Configuring quality settings {: id="configuring-quality-settings"}
+
+[Configure the quality settings](../repositories-configure/adjusting-quality-settings.md) of your project to define the gates (minimum acceptable values) and goals for quality metrics such as issues, duplication, complexity, and code coverage. These gates, goals, and metrics are always visible on Codacy's dashboard and [can be optionally enforced](#blocking-pull-requests) when merging pull requests.
+
+!!! tip
+    Depending on the initial quality of your repositories and on your current team practices, the default gates may be too strict. Consider initially limiting gates to new critical issues, new security issues, and considerable drops in code coverage.
+
+![Configuring quality settings](../repositories-configure/images/quality-settings.png)
+
+## 2. Activating the Git provider integration {: id="git-provider-integration"}
+
+1.  Activate the integration with your Git provider ([GitHub](../repositories-configure/integrations/github-integration.md), [GitLab](../repositories-configure/integrations/gitlab-integration.md), [Bitbucket](../repositories-configure/integrations/bitbucket-integration.md))
+1.  When activating the integration, enable the option **Status checks** (GitHub) or **Pull request status** (GitLab and Bitbucket)
+    -   This instructs Codacy to report to your Git provider whether the status checks on a pull requests are passing or failing, based on your quality settings
```

Comment:
I included some bits of layered information as sub-points for now, although this may not be the best format to present them. We may also simply use two sentences per point. Thoughts?